helm-service/controller/deployment_handler.go

```diff
@@ -257,3 +288,38 @@ func (h *DeploymentHandler) getFinishedEventDataForNoDeployment(eventData keptnv
 		EventData: eventData,
 	}
 }
+
+func (h *DeploymentHandler) validateDeploymentURI(deploymentURI string, maxRetries int) (int, error) {
```

Comment:
Can you please add a Unit test for this function? Also, I think this func does not have to be a pointer receiver for `DeploymentHandler`, as it does not use any of its properties